### ISSUE TITLE
Question about the hardware requirements for training the Laynet model

### ISSUE THREAD

**Issue by wodnl7** [lanchengzou/congPred#1]
This is an amazing project! I really enjoyed reading your paper and exploring the code. I have a question regarding the hardware requirements for training the Laynet model. In the paper, you mentioned that the training was done using 3090 GPU. However, from the Laynet model code, it seems that the model is quite large, and in addition to image data, it also requires graph data. Given the complexity and size of the model, I was wondering: Is a single 3090 GPU sufficient for training the model, or would it require multiple GPUs to handle the computational load?

**Comment by lanchengzou** [comment]
Thank you for your interest in our work!
We recommend to use at least two 3090 GPU for training.
With a single 3090, you maybe need to use a quite small batch size with gradient accumulation technique to update the parameters.

**Comment by hayaseyuukaa** [comment]
> This is an amazing project! I really enjoyed reading your paper and exploring the code. I have a question regarding the hardware requirements for training the Laynet model. In the paper, you mentioned that the training was done using 3090 GPU. However, from the Laynet model code, it seems that the model is quite large, and in addition to image data, it also requires graph data. Given the complexity and size of the model, I was wondering: Is a single 3090 GPU sufficient for training the model, or would it require multiple GPUs to handle the computational load?

Hello, have you successfully reproduced the paper? Thank you if you can provide the processed data set, or the method of processing
